src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OpprettOppgaveForBarnService.kt

```diff
@@ -0,0 +1,69 @@
+package no.nav.familie.ef.iverksett.oppgave
+
+import no.nav.familie.ef.iverksett.iverksetting.IverksettingRepository
+import no.nav.familie.ef.iverksett.iverksetting.domene.Iverksett
+import no.nav.familie.kontrakter.ef.søknad.Fødselsnummer
+import no.nav.familie.kontrakter.felles.oppgave.Oppgavetype
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+
+@Service
+class OpprettOppgaveForBarnService(private val oppgaveClient: OppgaveClient,
+                                   private val iverksettingRepository: IverksettingRepository) {
+
+    private val secureLogger = LoggerFactory.getLogger("secureLogger")
+
+    fun opprettOppgaverForAlleBarnSomFyller(innenAntallUker: Long) {
+        val behandlinger = iverksettingRepository.hentAlleBehandlinger()
+        behandlinger.forEach { iverksett ->
```

Comment:
Går vi ikke gjennom "for mange" her? Er kun siste behandling per fagsak som er aktuell - ikke alle foregående? 🤔 

Kanskje vi trenger fagsakId og "sistEndret" i `Iverksett`-tabellen? Eller hva tenker du @blommish ?

Reply:
Ja, vi skal ikke hente alle her
Jeg er litt usikker her på om denne funksjonaliteten burde ligge her eller i ef-sak

Reply:
Veldig bra, flyttet til ef-sak